gtfs/spec/en/reference.md

```diff
@@ -156,6 +164,7 @@ File: **Required**
 |  `wheelchair_boarding` | Enum | Optional | Indicates whether wheelchair boardings are possible from the location. Valid options are: <br><br>For parentless stops:<br>`0` or empty - No accessibility information for the stop.<br>`1` - Some vehicles at this stop can be boarded by a rider in a wheelchair.<br>`2` - Wheelchair boarding is not possible at this stop. <br><br>For child stops: <br>`0` or empty - Stop will inherit its `wheelchair_boarding` behavior from the parent station, if specified in the parent.<br>`1` - There exists some accessible path from outside the station to the specific stop/platform.<br>`2` - There exists no accessible path from outside the station to the specific stop/platform.<br><br> For station entrances/exits: <br>`0` or empty - Station entrance will inherit its `wheelchair_boarding` behavior from the parent station, if specified for the parent.<br>`1` - Station entrance is wheelchair accessible.<br>`2` - No accessible path from station entrance to stops/platforms. |
 |  `level_id` | ID referencing `levels.level_id` | Optional | Level of the location. The same level may be used by multiple unlinked stations.|
 |  `platform_code` | Text | Optional | Platform identifier for a platform stop (a stop belonging to a station). This should be just the platform identifier (eg. "G" or "3"). Words like “platform” or "track" (or the feed’s language-specific equivalent) should not be included. This allows feed consumers to more easily internationalize and localize the platform identifier into other languages. |
+| `area_id` | ID referencing `areas.area_id` | Optional | Identifies an area grouping of locations. Multiple rows in `stops.txt` may have the same `area_id`. |
```

Comment:
question: does this need a new file? `network_id` doesn't get a new file, and there's no additional information provided in `areas.txt` in this PR.

Reply:
I want to bring back this discussion. I agree with @paulswartz that forcing a redundant `areas.txt` is weird for the scope of this PR. Do we want to entertain the option of removing the file for the MVP of GTFS-Fares V2 and rely only on `stops.area_id` until a need for more complex zone systems using the full `areas.txt` emerges?